src/Actions/ValidatesConfiguration.php

```diff
@@ -30,12 +30,19 @@ public static function in($arguments): void
             throw new FileOrFolderNotFound('phpunit.xml');
         }
 
-        $configuration = Registry::getInstance()
-            ->get($arguments[self::CONFIGURATION_KEY])
-            ->phpunit();
+        $configuration = self::getConfiguration($arguments[self::CONFIGURATION_KEY]);
 
         if ($configuration->processIsolation()) {
             throw new AttributeNotSupportedYet('processIsolation', 'true');
         }
     }
+
+    private static function getConfiguration($item)
```

Comment:
Let's make sure we add static types here.

Reply:
👍

Reply:
Actually, not possible to add a return type, since the type different on different phpunit versions. PHPStan will definitely not be happy. One thing we could do is drop support for PHPUnit less than 9.3, which would mean this PR would have to wait until August, when PHPUnit 9.3 is released. I think this is fine, because PHPUnit 9.2 fails with a parse error due to https://github.com/sebastianbergmann/phpunit/pull/4374 only being included in PHPUnit 9.3.

Reply:
I think dropping PHPUnit less than 9.3 will be the way to go for this PR, actually.